src/Tools/babylon.promise.ts

```diff
@@ -22,36 +22,6 @@ module BABYLON {
         private _onRejected?: (reason: any) => void;
         private _rejectWasConsumed = false;
 
-        public get state(): PromiseStates {

```

Comment:
Why removing them?

Reply:
Not sure why they are needed as public. It is not part of the promise contract right?

Reply:
Right. I used them mostly for debugging purpose. We can remove them no problem 
I was more asking to understand if it was related to memory leak

Reply:
Yeah, I figured it was something like that. I removed them to see if they were the cause of any issues. I can put them back if you want.

Reply:
Nope this is fine like that